evalml/tests/data_checks_tests/test_target_leakage_data_check.py

```diff
@@ -254,37 +331,61 @@ def test_target_leakage_regression():
     assert leakage_check.validate(pd.DataFrame(), pd.Series()) == []
 
     y = pd.Series(
-        [0.4, 0.1, 2.3, 4.3, 2.2, 1.8, 3.7, 3.6, 2.4, 0.9, 3.1, 2.8, 4.1, 1.6, 1.2]
+        [
+            0.7,
+            0.8,
+            1.6,
+            2.8,
+            4.0,
+            5.1,
+            5.2,
+            5.9,
+            6.1,
+            6.4,
+            8.7,
+            9.5,
+            13.2,
+            13.7,
+            14.8,
+            15.2,
+            15.8,
+            19.0,
+            19.2,
+            19.6,
+            20.0,
+            20.6,
+            21.1,
+            23.3,
+            25.3,
+            25.8,
+        ]
     )
     X = pd.DataFrame()
     X["a"] = y * 3
     X["b"] = y - 1
     X["c"] = y / 10
-    X["d"] = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-    X["e"] = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o"]
-    X.ww.init(logical_types={"e": "categorical"})
+    X["d"] = np.zeros(len(y))
```

Comment:
Deleted column e because of this ticket: https://github.com/alteryx/woodwork/issues/1405